src/components/Picker/BasePicker/basePickerPropTypes.js

```diff
@@ -6,7 +6,7 @@ import * as Expensicons from '../../Icon/Expensicons';
 
 const propTypes = {
     /** A callback method that is called when the value changes and it received the selected value as an argument */
-    onChange: PropTypes.func.isRequired,
+    onInputChange: PropTypes.func.isRequired,
```

Comment:
Hmm we are still using `onChange` in like... all usages of Picker, so this will throw errors when using Picker outside of `Form`. I think we should rename `onChange` to `onInputChange` in all `Picker` usages.

Reply:
mmm, I remember someone ask me to change to `onInputChange`. But sure, will change back.